test/core/end2end/cq_verifier.c

```diff
@@ -71,17 +70,16 @@ typedef struct expectation {
 struct cq_verifier {
   /* bound completion queue */
   grpc_completion_queue *cq;
-  /* the root/sentinal expectation */
-  expectation expect;
+  /* expectation list */
+  expectation *first_expectation;
+  expectation *last_expectation;
```

Comment:
No worries.

It just occurred to me that there's not actually any reason we need to add the expectations to the end of the list, since the code that looks at the expectations doesn't care what order they're in.  So I've eliminated last_expectation and changed the code to add expectations to the start of the list instead.